Firmware/menu.cpp

```diff
@@ -73,6 +73,7 @@ void menu_start(void)
 		menu_top = lcd_encoder;
     menu_line = menu_top;
     menu_clicked = lcd_clicked(); // Consume click event
+	if (lcd_draw_update) lcd_frame_start();
```

Comment:
Would it make sense to call `lcd_frame_start()` in `lcd_update()`? Then `lcd_frame_start()` is only called in one place. LCD updates are always enabled in the menus.

Reply:
Hmmm, not really. Because we don't want to start a new frame if the entire screen isn't redrawn. I think if we did this in `lcd_update()`, then `lcd_frame_start()` gets called every 100ms, which is not what I want.

Reply:
Scrap my last comment. You were right. It is better to add the frame start in the lcd update function